client/internal/app/init.go

```diff
@@ -0,0 +1,72 @@
+package app
+
+import (
+	"client/internal/config"
+	"client/internal/logger"
+	"client/internal/roamer"
+	"client/internal/secret"
+	"client/internal/storage/repository"
+	"client/internal/syncer"
+	"client/internal/tui"
+	"client/internal/user"
+)
+
+type Storager interface {
+	user.UserAccessor
+	secret.SecretAccessor
+	syncer.StorageAccessor
+}
+
+type App struct {
+	tui tui.Tui // Морда в терминале
+
+	storage Storager // База
+	syncer  syncer.Item
+}
+
+func Make() (App, error) {
```

Comment:
App  - достатосно большая структура, которая к тому же может содержать сложные обхекты или примитивы сихнронизации, которые непредсказуемо могут повести себя при копировании. Поэтому я бы совертовал возвращать и далее работать со ссылкой на эту структуру.
```suggestion
func Make() (*App, error) {
```

Reply:
В целом я предполагал, что App может быть создан только однажды. Но да, лучше вернуть указатель. Структура большая